kogito-timer-quarkus/README.md

```diff
@@ -122,11 +208,18 @@ curl -H 'Content-Type:application/json' -H 'Accept:application/json' http://loca
 ### Cancel boundary timer instance 
 
 ```
-curl -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/timersOnTask/{uuid}'
+curl -X DELETE 'http://localhost:8080/timersOnTask/{uuid}'
 ```
 
 where `{uuid}` is the id of the given timer instance
 
+After executing the above commands you should see a log similar to
+
+* Curl Log
```

Comment:
I don't see that relevant. You can delete it.

Reply:
fixed @radtriste